recipe/meta.yaml

```diff
@@ -51,10 +48,13 @@ about:
   home: https://github.com/kubernetes-incubator/client-python
   license: Apache-2.0
   summary: The official Kubernetes python client.
+  description: Python client for kubernetes http://kubernetes.io/
   license_family: APACHE
   license_file: LICENSE
+  license_url: https://www.apache.org/licenses/LICENSE-2.0
   dev_url: https://github.com/kubernetes-incubator/client-python
```

Comment:
The `dev_url` has a redirect:
https://github.com/kubernetes-client/python

Reply:
done